shared/src/main/java/uk/gov/di/authentication/shared/helpers/EmailCheckMetricsHelper.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.di.authentication.shared.helpers;
+
+import uk.gov.di.authentication.shared.entity.JourneyType;
+import uk.gov.di.authentication.shared.entity.NotificationType;
+import uk.gov.di.authentication.shared.services.CloudwatchMetricsService;
+
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.di.authentication.shared.domain.CloudwatchMetricDimensions.ENVIRONMENT;
+
+public class EmailCheckMetricsHelper {
```

Comment:
Would be worth renaming this, maybe to `FraudCheckMetricsHelper`, as this will be used for both phone and email